sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/Diagnostics/AzureMonitorExporterEventSource.cs

```diff
@@ -4,17 +4,269 @@
 using System;
 using System.Diagnostics.Tracing;
 using System.Runtime.CompilerServices;
+using Azure.Monitor.OpenTelemetry.Exporter.Internals.ConnectionString;
```

Comment:
Why this is needed?

Reply:
`ConnectionVars` is passed into a `NonEvent` here:
This could probably be replaced.

https://github.com/Azure/azure-sdk-for-net/blob/fd30f1fede00d0daba59a6dc5d03bbb959b3adba/sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/Diagnostics/AzureMonitorExporterEventSource.cs#L43-L58